src/Ledger/Gov.lagda

```diff
@@ -138,3 +129,20 @@ _⊢_⇀⦇_,GOV⦈_ = ReflexiveTransitiveClosureᵢ _⊢_⇀⦇_,GOV'⦈_
 \caption{Rules for the GOV transition system}
 \label{defs:gov-rules}
 \end{figure*}
+
+The GOV transition system is now given as the reflexitive-transitive
+closure of the system GOV', described in
```

Comment:
Since we don't have anything that's called `GOV'` in the code I think it's better to not color it. Anything with syntax highlighting should refer specifically to a piece of Agda code.